src/loader.js

```diff
@@ -191,6 +191,10 @@ function logloads(loads) {
       }
 
       load = createLoad(name);
+      if(!load.requests)
+        load.requests = [];
+
+      load.requests.unshift({as: request, from: refererName});
```

Comment:
I couldn't see how to do it. I looked into the suggestion to trace-after-fail, but the trace itself fails and the after-trace code (which would report the trace) doesn't run in that case. I couldn't figure out how to get a sensible report out of a partial (i.e. failed) trace, even if the after-run code were used after a fail.

Possibly we could add a build option to enable load request tracking? Since the error report checks for the presence of the requests property, it'd fall back to the old behavior in its absence (that error message might refer to the option, though.)